src/dex/woo-fi/woo-fi.ts

```diff
@@ -565,6 +569,10 @@ export class WooFi extends SimpleExchange implements IDex<WooFiData> {
   }
 
   async updatePoolState(): Promise<void> {
+    if (this.isRefInfosEmpty) {
```

Comment:
This is a function, did you mean to call it here?

Reply:
Yes, thank you for catching it 🥇

Reply:
@ColonelJ 
Do we need to call getter property?

```
  get isRefInfosEmpty(): boolean {
    return Object.keys(this._refInfos).length === 0;
  }
```